docs/openstack/glance_adoption.md

```diff
@@ -22,22 +22,6 @@ This guide also assumes that:
 * Previous Adoption steps completed. Notably, MariaDB and Keystone
   should be already adopted.
 
-
-## Pre-check
```

Comment:
This assumed there's a running service on the src cloud w/o any issue: it's redundant because if the service is not running on the src cloud there's no point on following this guide, but if you have no images it should be empty but not fail, right?
Perhaps this could be turned into a generic statement on the status of the src service, but if it's not consistent w/ the rest of the doc, feel free to remove

Reply:
I missed the context here: the src service is already stopped at this time.
This is something that changed over the time, now I understand why you removed this section. Thanks, that makes a lot of sense now.